packages/serverless-contracts/src/contracts/apiGateway/features/mockHandlerInput.ts

```diff
@@ -0,0 +1,99 @@
+import { Context } from 'aws-lambda';
+import { JSONSchemaFaker, Schema } from 'json-schema-faker';
+import merge from 'lodash/merge';
+
+import {
+  getAPIGatewayEventRequestContextMock,
+  getAPIGatewayV2EventRequestContextMock,
+} from '@swarmion/serverless-helpers';
+
+import { GenericApiGatewayContract } from '../apiGatewayContract';
+import {
+  ApiGatewayEvent,
+  BodyType,
+  CustomRequestContextType,
+  HandlerCallback,
+  HandlerEventType,
+  HeadersType,
+  PathParametersType,
+  QueryStringParametersType,
+} from '../types';
+import {
+  ApiGatewayAuthorizerType,
+  ApiGatewayIntegrationType,
+} from '../types/constants';
+import { getHandlerContextMock } from '../utils/mockHandlerContext';
+
+export const getMockHandlerInput = <
+  Contract extends GenericApiGatewayContract,
+  IntegrationType extends ApiGatewayIntegrationType = Contract['integrationType'],
+  AuthorizerType extends ApiGatewayAuthorizerType = Contract['authorizerType'],
+  AdditionalArgs extends unknown[] = never[],
+  CustomEvent = HandlerEventType<
+    IntegrationType,
+    AuthorizerType,
+    PathParametersType<Contract>,
+    QueryStringParametersType<Contract>,
+    HeadersType<Contract>,
+    CustomRequestContextType<Contract>,
+    BodyType<Contract>
+  >,
+  Event = ApiGatewayEvent<IntegrationType, AuthorizerType>,
+  Callback = HandlerCallback<IntegrationType>,
+>(
+  contract: Contract,
+  partialEvent: Partial<CustomEvent> = {},
+  ...additionalArgs: AdditionalArgs
+): [Event, Context, Callback, ...AdditionalArgs] => {
+  const defaultBody =
+    contract.bodySchema !== undefined
+      ? JSONSchemaFaker.generate(contract.bodySchema as Schema)
```

Comment:
you could write a local `getDefault` function that takes the key of the contract as an argument and return the schema faker generate output, since you basically wrote the same thing 5 times